faq.md

```diff
@@ -112,162 +111,136 @@ It’s a component that was introduced recently. Essentially it’s a stateless
-PartitionedConsumer is composed of a set of regular consumers, one per partition. To have a single `receive()` abstraction, messages from all partitions are then pushed into a shared queue. 
-
-The thing is that the unacked message tracker works at the partition level.So when the timeout happens, it’s able to request redelivery for the messages and clear them from the queue when that happens,
-but if the messages were already pushed into the shared queue, the “clearing” part will not happen.
-
-- the only quick workaround that I can think of is to increase the “ack-timeout” to a level in which timeout doesn’t occur in processing
-- another option would be to reduce the receiver queue size, so that less messages are sitting in the queue
+- a quick workaround is to increase the “ack-timeout” to a level in which timeout doesn’t occur in processing
+- another option is to reduce the receiver queue size, so that less messages are sitting in the queue
 
 ### Can I use bookkeeper newer v3 wire protocol in Pulsar? How can I enable it?
-The answer is currently not, because we force the broker to use v2 protocol and that's not configurable at the moment.
+Currently, you cannot use bookkeeper v3 wire protocol in Pulsar. Because the broker is designed to use v2 protocol, and it is not configurable at the moment.
 
 ### Is "kubernetes/generic/proxy.yaml" meant to be used whenever we want to expose a Pulsar broker outside the Kubernetes cluster?
-Yes, the “proxy” is an additional component to deploy a stateless proxy frontend that can be exposed through a load balancer and that doesn’t require direct connectivity to the actual brokers. No need to use it from within Kubernetes cluster. Also in some cases it’s simpler to have expose the brokers through `NodePort` or `ClusterIp` for other outside producer/consumers.
-
-### Is there a way of having both authentication and the Pulsar dashboard working at same time?
-The key is that with authorization, the stats collector needs to access the APIs that require the credentials. That’s not a problem for stats collected through Prometheus but it is for the “Pulsar dashboard” which is where the per-topic stats are shown. I think that should be quite easy to fix.
+Yes, the “proxy” is an additional component to deploy a stateless proxy frontend that can be exposed through a load balancer, and that doesn’t require direct connectivity to the actual brokers. You do not need to use the proxy within Kubernetes cluster. In some cases, it’s simpler to expose the brokers through `NodePort` or `ClusterIp` for other producer/consumers outsides.
 
 ### How can I know when I've reached the end of the stream during replay?
```

Comment:
This FAQ needs to be updated. There is a method to know now.

Reply:
What's the method? how to conduct it?